src/webparts/todolist/ToDoListWebPart.ts

```diff
@@ -29,7 +29,8 @@ export default class ToDoListWebPart extends BaseClientSideWebPart<IToDoListWebP
         isDarkTheme: this._isDarkTheme,
         environmentMessage: this._environmentMessage,
         hasTeamsContext: !!this.context.sdks.microsoftTeams,
```

Comment:
hasTeamsContext, environmentMessage, isDarkTheme are not use in component. Need to delete this properties